notebooks/subdir.mk

```diff
@@ -1,20 +1,25 @@
 py_files := $(wildcard notebooks/py/*.py)
 ipynb_files := $(wildcard notebooks/*.ipynb)
 py_run_targets := $(addprefix run__, $(py_files))
+py_check_targets := $(addprefix check__, $(py_files))
 ipynb_validate_targets := $(addprefix validate__, $(ipynb_files))
 ipynb_regenerate_targets := $(addprefix regenerate__notebooks/, $(addsuffix .ipynb, $(notdir $(basename $(py_files)))))
 py_regenerate_targets := $(addprefix regenerate__notebooks/py/, $(addsuffix .py, $(notdir $(basename $(ipynb_files)))))
 PYTHON := python
 
 fast: $(ipynb_validate_targets)
 run_notebooks: $(py_run_targets)
+check_notebooks: $(py_check_targets)
 validate_notebooks: $(ipynb_validate_targets)
 regenerate_ipynb: $(ipynb_regenerate_targets)
 regenerate_py: $(py_regenerate_targets)
 
 $(py_run_targets): run__%.py :
 	[ -e $*.py.skip ] || $(PYTHON) $*.py
 
+$(py_check_targets): check__%.py :
+	grep $*.py .travis.yml
```

Comment:
Consider `grep -q` since otherwise we'll be spewing a bunch of stuff to the console.

Reply:
Pushed.